.eslintrc

```diff
@@ -110,12 +110,7 @@
         "allowRequiredDefaults": true
       }
     ],
-    "react/require-default-props": [
-      "error",
-      {
-        "forbidDefaultForRequired": false
-      }
-    ],
+    "react/require-default-props": "off",
```

Comment:
We might not need this rule anymore at all as our linter handles this for us here: https://github.com/Skyscanner/eslint-config-skyscanner/blob/main/index.js#L44-L53

Reply:
I had actually tried removing it and got a lot of lint errors so added it back 😢 
`forbidDefaultForRequired: true` we are actually using defaults for some required props.. not too sure why. But we can probably clean it up as we go and once all components have been migrated to TS we can remove it? 🤔

Reply:
Yeah let's just make sure we have a follow up task in the backlog to do that once we have migrated all :) and as we go through each component let's make sure no required props have any defaults to ensure we align with correct approaches